tests/simpleTest/simple_app_state.hpp

```diff
@@ -0,0 +1,139 @@
+// Concord
+//
+// Copyright (c) 2018-2022 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "assertUtils.hpp"
+#include "SimpleClient.hpp"
+#include "commonDefs.h"
+
+using namespace bftEngine;
+using namespace bft::communication;
+using namespace std;
+
+// NOLINTNEXTLINE(misc-definitions-in-headers)
+logging::Logger replicaLogger = logging::getLogger("simpletest.replica");
+
+#define test_assert_replica(statement, message)                                                                   \
+  {                                                                                                               \
+    if (!(statement)) {                                                                                           \
+      LOG_FATAL(replicaLogger, "assert fail with message: " << message); /* NOLINT(bugprone-macro-parentheses) */ \
+      ConcordAssert(false);                                                                                       \
+    }                                                                                                             \
+  }
+
+// The replica state machine.
+class SimpleAppState : public IRequestsHandler {
+ private:
+  uint64_t client_to_index(NodeNum clientId) { return clientId - numOfReplicas; }
```

Comment:
please change all the names of function in camelcase as below:
clientToIndex
getLastStateValue
getLastStateNum
setLastStateValue
setLastStateNum

Reply:
Fixed